exchanges/gateio/gateio_wrapper.go

```diff
@@ -2112,58 +2124,56 @@ func (g *Gateio) GetFuturesContractDetails(ctx context.Context, item asset.Item)
 		return resp, nil
 	case asset.DeliveryFutures:
 		var resp []futures.Contract
-		for k := range settlementCurrencies {
-			contracts, err := g.GetAllDeliveryContracts(ctx, settlementCurrencies[k])
+		contracts, err := g.GetAllDeliveryContracts(ctx, currency.USDT)
+		if err != nil {
+			return nil, err
+		}
+		contractsToAdd := make([]futures.Contract, len(contracts))
+		for j := range contracts {
```

Comment:
Since you're touching Could we start to use `i` consistently [for loop statement index counters](https://go.dev/ref/spec#For_statement) please :pleading_face: :pray:

Reply:
Despite using i for years, I don't really mind if people want to personalise something as insignificant as index counter vars. I'm infatuated with the letters x, y and z (though even moreso on x) for reference😆 So, will pass